Apps/Medication/src/Models/HNMessage.cs

```diff
@@ -67,7 +67,7 @@ public HNMessage(T response, ResultType result, string resultMessage)
         /// <summary>
         /// Gets or sets the response message.
         /// </summary>
-        public T Message { get; set; }
+        public T? Message { get; set; }
```

Comment:
if the message is null, would that not mean that something went bad?

Reply:
We have two constructors that set the Message to null.